application/frontend/controllers/MfaController.php

```diff
@@ -90,7 +96,7 @@ public function actionVerify(int $id)
 
         // Strip spaces from $value if string
         if (is_string($value)) {
-            $value = preg_replace('/\D/', '', $value);
+            $value = preg_replace('/\s+/', '', $value);
```

Comment:
It may be worth adding a comment here, either saying what `\s` represents or merely pointing to the documentation:
https://www.php.net/manual/en/regexp.reference.character-classes.php

Reply:
The comment was already there above the `if` statement.

Reply:
Looks like this should be reverted, since `$value` apparently is only supposed to be a string for TOTP. For WebAuthn, `$value` should already be a key/value array.